sandbox/anvi-script-get-coverage-from-bam

```diff
@@ -36,7 +36,7 @@ class ComputeCoverage(object):
         self.sanity_check()
         self.f = self.init_output()
 
-        self.bam = pysam.Samfile(self.args.bam_file, 'rb')
+        self.bam = bamops.BAMFileObject(self.args.bam_file, 'rb')
```

Comment:
does this mean this previous version of this line will explode in tagged `v6.2.`? :) asking for a friend.

Reply:
Fortunately, no. `Samfile` is a deprecated but functioning class as of pysam 0.15.4... 

But I'm realizing that the `requirements.txt` has 0.15.2.... What should we do?

Reply:
Thanks for the catch. Fixed in 5f6632c